### PR TITLE
remove extra `Engine` after class name

### LINKED ISSUE #890: Incorrect cache engine name in home page
This is a (multiple allowed):

* [x] bug

* CakePHP Application Skeleton Version: 4.3.1

### What you did

Open the home page.

### What happened

In Filesystem section:

> The Cake\Cache\Engine\FileEngineEngine is being used for core caching. To change the config edit config/app.php

### What you expected to happen

> The Cake\Cache\Engine\FileEngine is being used for core caching. To change the config edit config/app.php

### Information

This displays `className` of `_cake_core_` cache config, followed by "Engine".
Previously `className` was short name like `File`, but changed to FQCN at 4f7588486e34df7da712449f137fb19dd4000622.
I think it's good to display FQCN even when configured with short name, though it requires instantiating the engine.

### PR DESCRIPTION
**Pull request by saeideng**
Close #890

### CODE REVIEW

File: templates/Pages/home.php [hunk 1]
```diff
@@ -147,7 +147,7 @@
 
                         <?php $settings = Cache::getConfig('_cake_core_'); ?>
                         <?php if (!empty($settings)) : ?>
-                            <li class="bullet success">The <em><?= $settings['className'] ?>Engine</em> is being used for core caching. To change the config edit config/app.php</li>
+                            <li class="bullet success">The <em><?= $settings['className'] ?></em> is being used for core caching. To change the config edit config/app.php</li>
```
Comment (dereuromark) [inline templates/Pages/home.php:150]: h() would also be good to add

Comment (saeideng) [reply templates/Pages/home.php:150]: `$settings['className']` comes from config file , not user input

Comment (dereuromark) [reply templates/Pages/home.php:150]: Still good practice imo

Comment (markstory) [reply templates/Pages/home.php:150]: I can add it separately. We should have it if only to model safe template authoring.

**Pull request merged**